### ISSUE TITLE
Games owned by multiple developers

### ISSUE THREAD

**Issue by tsbandit** [leafo/itch.io#206]
I often develop small games with small teams. The team members are different every time. We would each like to have our own separate accounts, and then submit our game with joint ownership.

**Comment by leafo** [comment]
Are you looking for more functionality than what is provided with game admins: http://blog.itch.io/post/77095066632/let-collaborators-edit-your-game-pages-with-the

**Comment by tsbandit** [comment]
Sorry, I failed to discover the game admins feature until now. We will try it out and then let you know if we need more functionality.

**Comment by robertpfeiffer** [comment]
More functionality: It would be neat if the game could appear on a contributor's profile page.

**Comment by carniblood** [comment]
Also when participating in a game jam, it's important to have the link "View their submission" when commenting on other participants' submission, but this link doesn't appear for contributors, only for submitter.
Would it be possible to change this? thanks!

**Comment by iPruch** [comment]
I'm with @carniblood. We recently hosted a game jam on itch.io and people were asking to co-own games. Adding people as administrators makes it possible to edit the game, but the URL still begins with the uploader's nickname and the HTML title is "GameTitle by Uploader'sNickname", so this feature would be nice. Thanks!

**Comment by fasterthanlime** [comment]
> but the URL still begins with the uploader's nickname

Just curious, how would you expect co-ownership to work, as far as URLs are concerned? Would you 'create an organization', a-la GitHub ?

**Comment by iPruch** [comment]
@fasterthanlime or just two URLs that point to the same page, each beginning with each owner's nickname. But I don't know which technical difficulties would that imply.

**Comment by andrecosta** [comment]
The recent Ludum Dare event has brought this issue to light again for me and my team.

@leafo The game admins functionality is great but we would like to see project contributors have more visibility options than what it currently provides. My first thought was allowing each contributor to have their own URL to the same project (as @iPruch mentioned) but this may impact SEO and the project's identity.

@fasterthanlime Allowing the creation of "organizations/teams", which individual users could belong to, sounds good but contributors who participate in many different projects with different people may not find this ideal, especially if the team has the same URL scheme as individual users (since it would take up a whole username).

Also, if the objective is to have a single URL, one could already achieve the same effect by just creating a new "shared" account and keep the games on there. In that case, an option to transfer ownership of current games would be needed (I don't know if this is currently possible). Even so, I would prefer not to do this since it also takes up a username.

As an alternative, I would suggest that we have some kind of project-centric URLs (with their own naming structure), which link directly to the project. The contributors would still be visible on the game's details section (although they could be emphasised even more than they are now).

For example, something like:
http://itch.io/10203/project-name

Even if translating to a possible "teams" implementation, it could be similar:
http://itch.io/team-name/project-name

Of course, this would also technically be a redundant URL of the user's own individual project URL, unless you only allow one of them to be active. I'm surely divided on how to approach this.

**Comment by Guard13007** [comment]
When setting up something, I would add the suggestion of having only one URL as "the" URL but others formatted in a way in which they would have been pointing to the same thing are redirected.

**Comment by LLucile** [comment]
At least a feature really needed is a way to show those games on each collaborator profiles without using a collection, and as a game jam submission too.
This way, teams could always create a new account just for this game, set each of the members as collaborators of the project and still get the project to show up on their itch.io account (all pointing the project's owner URL) so that it can look like a "organization" submission if they want to.

**Comment by Glanzer** [comment]
Would it make sense to combine this with https://github.com/itchio/itch.io/issues/521 ?

**Comment by dos1** [comment]
@Glanzer Personally I don't really care about #521, but care a lot about this one.

It's ridiculous that we have to either fight on who's uploading the game, duplicate games or abuse collections (which isn't really an acceptable solution) each time we jam with other people, especially on a platform seemingly focused on jams. Please, add a simple checkbox "show on my profile" - currently game admins feature is almost useless, and this simple change would change everything :( Waiting for it for years now...

**Comment by Glanzer** [comment]
@dos1 I completely agree. Just thought that attaching relevant issues might add to overall priority. Also it shows that the whole team-aspect is a little underserved in most if not every aspect of the platform. The portfolio-aspect is the biggest problem here, tho.

Revenue-splitting would ideally be done by contract and a single legal entity attached to a specific user-account, but I hope it's apparent that this approach, although legit, would run counter the intention of the platform. (at least in the way I understand it)

**Comment by Guard13007** [comment]
While I haven't worked on anything team-based yet, I just wanted to put
another comment to say I believe this should be a higher priority.

**Comment by leafo** [comment]
You can now select game's you're an admin for to display on your profile page on the top area. You no longer need to use a collection to display them on your profile. Tell me if you encounter any issues

**Comment by Glanzer** [comment]
Awesome! Thanks.

**Comment by fede-Raider** [comment]
> You can now select game's you're an admin for to display on your profile page on the top area. You no longer need to use a collection to display them on your pro

Sorry for the questions, but how?

**Comment by TangentFoxy** [comment]
@egordorichev There is another issue, #521 for that.

(Shouldn't this issue be closed now?)

**Comment by afk-mario** [comment]
It would be nice that on the top right corner section of the game in the "badges" section it shows a list of all the contributors like the one that says `View all by...` but one for each contributor

**Comment by dill0wn** [comment]
Thanks to this thread I found out about the Project Admins feature.

This looks like it'll work well for my jam team.

One minor thing I have run into is only the project owner can submit the project to Jams. I think this is ok, just something to watch out for.

**Comment by coldacid** [comment]
How about moving a game from one account to another account?
